plugins/UsersManager/API.php

```diff
@@ -1379,4 +1390,71 @@ private function getRoleAndCapabilitiesFromAccess($access)
         }
         return [$roles, $capabilities];
     }
+
+    private function sendEmailChangedEmail($user, $newEmail)
```

Comment:
could we maybe mostly reuse the same code for `sendEmailChangedEmail` and `sendPasswordChangedEmail ` and also it's templates and only have different mail subject and also different leading sentence in the email? not too important though.

Reply:
Thought about it, but since there were just two emails, and the code isn't really complicated (just setting things on the view & Mail instance), it didn't seem that important. Maybe if there are more changes to a user that we should notify them about?